x-pack/platform/plugins/private/monitoring/public/alerts/large_shard_size_alert/get_description_fields.test.ts

```diff
@@ -81,4 +83,28 @@ describe('shard size getDescriptionFields', () => {
     expect(mockPrebuildField).toHaveBeenCalledWith(['logs-*']);
     expect(result).toEqual([mockReturnValue]);
   });
+
+  it('should call prebuildField with custom query in array', () => {
+    const mockRule = {
+      params: {
+        indexPattern: 'logs-*',
+        filterQueryText: 'some: query',
+      },
+    } as unknown as Rule<RuleParams>;
+
+    const mockReturnValue = { type: 'index_pattern', value: 'logs-*' };
+    const mockCustomQueryReturnValue = { type: 'custom_query', value: 'some: query' };
+    mockPrebuildField.mockReturnValue(mockReturnValue);
+    mockPrebuildCustomQuery.mockReturnValue(mockCustomQueryReturnValue);
+
+    const result = getDescriptionFields({
```

Comment:
It seems that this is coming from `x-pack/platform/plugins/private/monitoring/public/alerts/large_shard_size_alert/index.tsx` and not from `x-pack/platform/plugins/private/monitoring/public/alerts/large_shard_size_alert/get_description_fields.ts`.